data_diff/dbt.py

```diff
@@ -31,14 +32,26 @@ def import_dbt():
 from .utils import run_as_daemon, truncate_error
 from . import connect_to_table, diff_tables, Algorithm
 
-RUN_RESULTS_PATH = "/target/run_results.json"
-MANIFEST_PATH = "/target/manifest.json"
-PROJECT_FILE = "/dbt_project.yml"
-PROFILES_FILE = "/profiles.yml"
+RUN_RESULTS_PATH = "target/run_results.json"
+MANIFEST_PATH = "target/manifest.json"
+PROJECT_FILE = "dbt_project.yml"
+PROFILES_FILE = "profiles.yml"
 LOWER_DBT_V = "1.0.0"
 UPPER_DBT_V = "1.5.0"
 
 
+# https://github.com/dbt-labs/dbt-core/blob/c952d44ec5c2506995fbad75320acbae49125d3d/core/dbt/cli/resolvers.py#L6
+def default_project_dir() -> Path:
```

Comment:
we should update the help text here https://github.com/datafold/data-diff/blob/d2d78495e650fd336c762f164315cb7e1b706074/data_diff/__main__.py#L208

```
      --dbt-profiles-dir PATH         Override the default dbt profile location
                                      (~/.dbt).
      --dbt-project-dir PATH          Override the dbt project directory.
                                      Otherwise assumed to be the current
                                      directory.
```

Reply:
Great catch @dlawin !

Added updates to the help text in commit https://github.com/datafold/data-diff/pull/415/commits/098565af43e7994b900e31e5e1ea594b78644866.

Copied verbatim from [here](https://github.com/dbt-labs/dbt-core/blob/0ef9931d197c996ccd4e3caf641729176eceda99/core/dbt/cli/params.py#L243-L257). Can adjust the language if you want.